src/app/iteration/iteration.component.html

```diff
@@ -24,7 +24,10 @@
       <li class="active" *ngFor="let iteration of currentIterations">
         <div class="iteration-subheader">
           <div class="iteration-name truncate">
-            <strong>{{iteration.attributes.name}}</strong>
+            <strong tooltip="{{iteration.attributes.name}}"
```

Comment:
Yes, we need tooltip. See the SJ comment here https://github.com/fabric8io/fabric8-planner/issues/831